packages/core/src/ceramic.ts

```diff
@@ -870,31 +870,33 @@ export class Ceramic implements CeramicApi {
 
     const index = {}
     const walkNext = async (node: TrieNode, streamId: StreamID | CommitID) => {
+      const queryAtTime = query.opts?.atTime ? query.opts?.atTime : query.atTime
+      const opts = queryAtTime ? { atTime: queryAtTime, ...query.opts } : {... query.opts}
```

Comment:
If we use query.opts here rather than {... query.opts} directly we lose the SyncOpts that are not explicitly part of LoadOpts and the sync tests fail.

Reply:
Hmm sorry I don't understand.  `{...query.opts}` should create an object that is totally identical to `query.opts`. It's just doing a shallow copy of all the fields.  I don't see how it could change the results of the test at all.  Maybe I'm missing something?

Reply:
From what I understand: 

```
LoadOpts extends SyncOpts. Therefore the schema to LoadOpts is:
{
 atTime
 sync
 syncTimeoutSeconds
}
```

`{... query.opts}` should copy all those fields.
`{query.opts}` would drop all of the original opts Sync fields on the floor.

Is there an assumption here that is incorrect?

Reply:
Correction: 

```
{
 loadOpts
 sync
 syncTimeoutSeconds
}
```

Reply:
note the code suggestion is replacing `{... query.opts}` with `query.opts` not with `{query.opts}`.  Just using the opts object directly.  There should be no fields dropped as it's just using the object as-is.